Updates for MinIO RELEASE.2023-04-07T05-28-58Z (#823)
minio/docs: source/administration/batch-framework.rst

Replicate
~~~~~~~~~

Use the ``replicate`` job type to create a batch job that replicates objects from one MinIO deployment to another MinIO location.
At least one of the deployment locations, either the source or the target, must be ``local``.
The definition file can limit the replication by bucket, prefix, and/or filters to only replicate certain objects.

.. versionchanged:: MinIO RELEASE.2023-04-07T05-28-58Z

   You can replicate from a remote MinIO deployment to the local deployment that runs the batch job.

> For example, you can use a batch job to perform a one-time replication sync of objects from ``minio-alpha/invoices/`` to ``minio-baker/invoices``.

Reviewer: Since before we're talking about "local" and "remote", maybe label one of these as the "local" and one as the "remote" and note that it works in both directions.

spitballing

"For example, you can create a batch job that performs a one-time "push" replication sync on the "local" ``minio-alpha/invoices`` deployment from the remote ``minio-baker/invoices`` resource.
The batch job could also do the reverse, "pulling" from the remote resource into the local."

Lots of quote-y things so edit at will.

Create a basic ``replicate`` job definition file you can edit with :mc:`mc batch generate`.

.. literalinclude:: /includes/code/replicate.yaml
   :language: yaml

Key Rotate
~~~~~~~~~~

.. versionadded:: MinIO RELEASE.2023-04-07T05-28-58Z

Use the ``keyrotate`` job type to create a batch job that cycles the :ref:`sse-s3 or sse-kms keys <minio-sse-data-encryption>` for encrypted objects.

> The YAML configuration allows you to use optional filters to rotate the keys for a specific set of objects by creation date, tags, metadata, or kms key.

Reviewer: ```suggestion
The YAML configuration supports filters to restrict key rotation to a specific set of objects by creation date, tags, metadata, or kms key.
```

dewording
